client/js/widgets/scoreboard.js

```diff
@@ -0,0 +1,287 @@
+          return pa < pb ? -1 : 1;
+        });
+      if(!this.get('sortAscending'))
+        pScores = pScores.reverse();
+
+      // Get player colors if needed
+      if(usePlayerColors)
+        for (let i=0; i < seats.length; i++)
+          colors.push(seats.filter(x=> x.get('player') == pScores[i][0])[0].get('color'));
+    } else if(typeof seats == 'object') { // Display team scores
+      usePlayerColors = false; // No predefined colors for teams
+      let i = 0;
+      for (const team in seats) {
+        // Get array of (arrays of) seat scores.
+        const seatScores = seats[team].map(w =>  asArray(w.get(scoreProperty)));
+
+        // Make all score arrays for this the same length, then add them element-by-element
+        const n = seatScores.reduce((max, xs) => Math.max(max, xs.length), 0);
+        pScores[i] = Array(n).fill(0).map((_,i) => this.getTotal(seatScores.map(xs => xs[i] || 0)));
+        pScores[i] = pScores[i].concat(Array(numRounds).fill('')).slice(0,numRounds);
+
+        // Add totals and team name
+        if(showTotals)
+          pScores[i].push(this.getTotal(pScores[i]));
+        pScores[i].unshift(team);
+        i++
+      }
+    } else // 'seats' property is not null, array, or object, ignore it.
+      return null;
+
+
+    // Fill empty player names with 'None'
+    for (let i=0; i < pScores.length; i++)
+      if(pScores[i][0]=='')
+        pScores[i][0] = 'None';
+
+    // Create round name headers
+    if(Array.isArray(rounds))
+      rounds = rounds.concat(Array(numRounds).fill('')).slice(0,numRounds);
+    else
+      rounds = [...Array(numRounds).keys()].map(i => i+1);
+    if(showTotals)
+      rounds.push('Totals');
+    rounds.unshift(this.get('roundLabel'));
+
+    // Finally, build the table
+    if(!this.tableDOM) {
+      this.tableDOM = document.createElement('table');
+      this.domElement.appendChild(this.tableDOM);
+    } else {
+      this.tableDOM.innerHTML = '';
+    }
+
+    this.tableDOM.style.height = this.get('height') + 'px';
+    this.tableDOM.style.width = this.get('width')+ 'px';
+    let numRows;
+    let numCols;
+    let currentRound = parseInt(this.get('currentRound'));
+
+    if (isNaN(currentRound) || currentRound <1 || currentRound > numRounds)
+      currentRound = null;
+
+    if(this.get('playersInColumns')) {
+      // Compute total number of rows and columns in table
+      numCols = pScores.length + 1;
+      numRows = numRounds + 1 + (showTotals ? 1 : 0);
+
+      // Add header row
+      const names = pScores.map(x => x[0]);
+      names.unshift(this.get('roundLabel'));
+      this.tableDOM.innerHTML += '<tbody></tbody>';
+      const tr = this.addRowToTable($('tbody', this.tableDOM), names, 'td');
+      if(usePlayerColors)
+        for (let c=0; c<pScores.length; c++ ) {
+          tr.cells[c+1].style.backgroundColor = colors[c];
+          tr.cells[c+1].style.color = 'white';
```

Comment:
Shouldn't this be black for light player colors? There are pretty simple formulas for calculating the lightness online.

Alternatively you could add an outline using `text-shadow`.